files/scripts/reports-cronjob.sh

```diff
@@ -26,7 +27,7 @@ source "${GEN3_HOME}/gen3/gen3setup.sh"
 
 help() {
   cat - <<EOM
-Use: bash ./reports-cronjob.sh vpc=YOUR_VPC_NAME
+Use: bash ./reports-cronjob.sh vpc=YOUR_VPC_NAME [ns_name=K8S_NAMESPACE]
```

Comment:
can you remove the brackets?

Reply:
oh, that is just to indicate it is optional :)

Reply:
👍